compiler/passes/scopeResolve.cpp

```diff
@@ -1918,7 +1959,26 @@ static bool lookupThisScopeAndUses(BaseAST* scope, const char * name,
         forv_Vec(ModuleSymbol, mod, *modules) {
           if (mod) {
             if (Symbol* sym = inSymbolTable(mod->block, name)) {
-              symbols.push_back(sym);
```

Comment:
It appears to me that lines 1962 - 1981 are exactly the same as 1893 - 1909.  If I am reading that correctly, then I am "ok" with this but wonder if there's an opportunity to share a common implementation?